src/TONClient.js

```diff
@@ -364,10 +364,17 @@ export class TONClientError {
             && (error.code === code);
     }
 
-    static isContractError(error: any, exitCode: number): boolean {
+    static isContractError(error: any, exitCode: number, originalError: boolean): boolean {
```

Comment:
What does `original_error` means? Why we need this additional flag?

Reply:
It is absolutely necessary to remove this flag and introduce new function `isContractErrorTraTaTa`.